routes/company-routes.js

```diff
@@ -1,105 +0,0 @@
-const express = require("express");
-const router = express.Router();
-const Review = require("../models/review");
-const User = require("../models/user");
-const Company = require("../models/company");
-const { isLoggedIn, isLoggedOut } = require("../middleware/route-guard");
-const { averagesObject, getOverallReviewScore } = require("../public/javascripts/averages");
-
-// Created an individual company page where you could see reviews left for that company. It require accessing the Company and User collections, sorting and deep-populating the query before calculating the averages and displaying all reviews. 
-
-router.get("/:id", (req, res, next) => {
-  const { id } = req.params;
-  let comp;
-  Company.findById(id)
-    .then((company) => {
-      comp = company;
-      return Review.find({ companyBeingReviewed: company })
-        .sort({createdAt:-1})
-        .populate([{ path: "companyBeingReviewed" }, { path: "owner" }]);
-    })
-    .then((reviews) => {
-      reviews.forEach(review => console.log(review.owner));
-      const averagesObj = averagesObject(reviews);
-      const overall = getOverallReviewScore(reviews);
-      res.render("companyprofile", { user: req.session.currentUser, reviews, averagesObj, noReviews: reviews.length, overall, comp });
-    })
-    .catch((err) => console.log(err));
-});
-
-// CRUD routes to be able to viwe and add companies to DB using Postman.
-
-router.get("/", (req, res, next) => {
-  Company.find()
-    .then((companies) => res.send(companies))
-    .catch((err) => console.log(err));
-});
-
-router.post("/new", (req, res, next) => {
-  const { name, logo, intro, pricing, type, usp, website } = req.body;
-  if (!name || !logo || !intro || !pricing || !type || !usp || !website) {
-    res.send("missing something");
-    return res.render("reviews/new", {
```

Comment:
Does this work? Again I'm not entirely sure as I've not worked in this in a while but I _think_ you can't do a `res.render` after a `res.send`. Not sure though...